qemu/tests/virtio_console.py

```diff
@@ -578,6 +578,18 @@ def test_loopback():
             logging.error(msg)
             test.fail(msg)
 
+    def get_virtio_port_by_name(vm, name):
```

Comment:
You don't need this function, you can simply get the device by `vm.devices.get_by_params({"name": name})`